http_client.rst

```diff
@@ -731,6 +731,59 @@ The ``RetryableHttpClient`` uses a
 decide if the request should be retried, and to define the waiting time between
 each retry.
 
+Retry Over Different URIs
+.........................
+
+.. versionadded:: 6.3
+
+    The multiple ``base_uri`` feature was added in Symfony 6.3.
+
+The ``RetryableHttpClient`` can be configured to use multiple base URIs. This
+feature provides increased flexibility and reliability for making HTTP
+requests. Pass an array of base URIs as option ``base_uri`` when making a 
+request::
+
+    $response = $client->request('GET', 'foo-bar', [
```

Comment:
```suggestion
    $response = $client->request('GET', 'get-articles', [
```

same for the other foo-bar example

Reply:
Oh thanks, I had it changed to `some-page` but the change was lost during rebase.

Reply:
some-page is better than get-articles